src/data/super_gameboy.asm

```diff
@@ -1024,308 +1042,184 @@
+    db   $01, $14, $D0, $14, $D1, $14, $8A, $14, $8A, $14, $8A, $14, $D8, $14, $D8, $14, $D8, $14, $D8, $14, $D8, $14, $D8, $14, $D8, $14, $D8, $14, $D8, $14, $D2, $10, $D3, $10, $D4, $14, $D5, $14, $D6, $14, $D8, $14, $D8, $14, $D7, $14, $D8, $10, $D8, $10, $D9, $10, $01, $14, $01, $14, $01, $14, $01, $14, $01, $14, $01, $14
+    db   $01, $14, $01, $14, $DA, $10, $CF, $14, $DB, $10, $DC, $10, $DD, $10, $8A, $14, $8A, $14, $8A, $14, $8A, $14, $8A, $14, $DE, $10, $DF, $10, $E0, $10, $E1, $14, $E2, $14, $E3, $14, $E4, $14, $8A, $14, $8A, $14, $E5, $14, $E6, $14, $8A, $10, $8A, $10, $B1, $10, $01, $14, $01, $14, $01, $14, $01, $14, $01, $14, $01, $14
+    db   $01, $14, $01, $14, $01, $14, $01, $14, $E7, $10, $E8, $10, $E9, $14, $EA, $14, $E8, $10, $E9, $14, $EA, $14, $EB, $14, $EC, $14, $ED, $14, $EE, $14, $EF, $14, $F0, $14, $F1, $14, $F2, $14, $F8, $14, $F0, $14, $F3, $14, $F4, $14, $B8, $10, $B9, $10, $01, $14, $01, $14, $01, $14, $01, $14, $01, $14, $01, $14, $01, $14
+    db   $01, $14, $01, $14, $01, $14, $01, $14, $01, $14, $F5, $10, $F6, $14, $F7, $14, $F6, $14, $F7, $14, $F8, $14, $F6, $14, $F7, $14, $F8, $14, $F9, $14, $FA, $14, $FB, $14, $FB, $14, $80, $14, $80, $54, $F4, $14, $B8, $10, $C1, $10, $C2, $10, $01, $14, $01, $14, $01, $14, $01, $14, $01, $14, $01, $14, $01, $14, $01, $14
+    db   $01, $14, $01, $14, $01, $14, $01, $14, $01, $14, $01, $14, $C2, $50, $C1, $50, $B8, $50, $FC, $14, $FD, $14, $FC, $14, $FD, $14, $FC, $14, $8A, $10, $8A, $10, $8A, $10, $8A, $10, $8A, $10, $8A, $10, $A9, $10, $C2, $10, $01, $14, $01, $14, $01, $14, $01, $14, $01, $14, $01, $14, $01, $14, $01, $14, $01, $14, $01, $14
+   ; unseen rows
+    db   $01, $14, $01, $14, $01, $14, $01, $14, $01, $14, $01, $14, $01, $14, $01, $14, $C2, $50, $A9, $50, $8A, $10, $8A, $10, $8A, $10, $8A, $10, $8A, $10, $8A, $10, $8A, $10, $8A, $10, $8A, $10, $8A, $10, $B1, $10, $01, $14, $01, $14, $01, $14, $01, $14, $01, $14, $01, $14, $01, $14, $01, $14, $01, $14, $01, $14, $01, $14
+    db   $01, $14, $01, $14, $01, $14, $01, $14, $01, $14, $01, $14, $01, $14, $01, $14, $01, $14, $B1, $50, $8A, $10, $8A, $10, $8A, $10, $8A, $10, $8A, $10, $8A, $10, $8A, $10, $8A, $10, $B8, $10, $B9, $10, $01, $14, $01, $14, $01, $14, $01, $14, $01, $14, $01, $14, $01, $14, $01, $14, $01, $14, $01, $14, $01, $14, $01, $14
+    db   $01, $14, $01, $14, $01, $14, $01, $14, $01, $14, $01, $14, $01, $14, $01, $14, $01, $14, $01, $14, $B9, $50, $B8, $50, $8A, $10, $8A, $10, $8A, $10, $8A, $10, $A9, $10, $AA, $10, $C2, $10, $01, $14, $01, $14, $01, $14, $01, $14, $01, $14, $01, $14, $01, $14, $01, $14, $01, $14, $01, $14, $01, $14, $01, $14, $01, $14
+    db   $01, $14, $01, $14, $01, $14, $01, $14, $01, $14, $01, $14, $01, $14, $01, $14, $01, $14, $01, $14, $01, $54, $C2, $50, $C1, $50, $A9, $50, $A9, $10, $FE, $10, $01, $14, $01, $14, $01, $14, $01, $14, $01, $14, $01, $14, $01, $14, $01, $14, $01, $14, $01, $14, $01, $14, $01, $14, $01, $14, $01, $14, $01, $14, $01, $14
+
```

Comment:
Now that you identified the content of each data block, do you think it would make sense to move these arrays into separate files?

Something like:
```
data/super_gameboy/tiles.asm
data/super_gameboy/tilemap.asm
data/super_gameboy/palettes.asm
```